src/formio/components/AddressNL.js

```diff
@@ -324,67 +330,53 @@ const FormikAddress = ({required, setFormioValues, deriveAddress}) => {
-          <TextField
-            name="houseLetter"
-            label={
-              <FormattedMessage
-                description="Label for addressNL houseLetter input"
-                defaultMessage="House letter"
-              />
-            }
+    <div
+      className={
+        useColumns
+          ? 'openforms-addressnl openforms-addressnl--double-column'
+          : 'openforms-addressnl openforms-addressnl--single-column'
+      }
+    >
+      <PostCodeField required={required} autoFillAddress={autofillAddress} />
+      <HouseNumberField required={required} autoFillAddress={autofillAddress} />
+      <TextField
+        name="houseLetter"
+        label={
+          <FormattedMessage
+            description="Label for addressNL houseLetter input"
+            defaultMessage="House letter"
           />
-        </div>
-        <div className="column column--span-6 openforms-form-field-container">
-          <TextField
-            name="houseNumberAddition"
-            label={
-              <FormattedMessage
-                description="Label for addressNL houseNumberAddition input"
-                defaultMessage="House number addition"
-              />
-            }
+        }
+      />
+      <TextField
+        name="houseNumberAddition"
+        label={
+          <FormattedMessage
+            description="Label for addressNL houseNumberAddition input"
+            defaultMessage="House number addition"
           />
-        </div>
-        {deriveAddress && (
-          <>
-            <div className="column column--span-6 openforms-form-field-container">
-              <TextField
-                name="city"
-                label={
-                  <FormattedMessage
-                    description="Label for addressNL city read only result"
-                    defaultMessage="City"
-                  />
-                }
-                disabled
-              />
-            </div>
-            <div className="column column--span-6 openforms-form-field-container">
-              <TextField
-                name="streetName"
-                label={
-                  <FormattedMessage
-                    description="Label for addressNL streetName read only result"
-                    defaultMessage="Street name"
-                  />
-                }
-                disabled
-              />
-            </div>
-          </>
-        )}
-      </div>
+        }
+      />
+      <TextField
+        name="city"
+        label={
+          <FormattedMessage
+            description="Label for addressNL city read only result"
+            defaultMessage="City"
+          />
+        }
+        disabled
+      />
+      <TextField
+        name="streetName"
+        label={
+          <FormattedMessage
+            description="Label for addressNL streetName read only result"
+            defaultMessage="Street name"
+          />
+        }
+        disabled
+      />
```

Comment:
I think it makes more sense to swap the order of these two fields? So first street, then city.